utils/scarb-test-support/src/project_builder.rs

```diff
@@ -24,6 +24,7 @@ pub struct ProjectBuilder {
     src: HashMap<Utf8PathBuf, String>,
     deps: Vec<(String, Value)>,
     dev_deps: Vec<(String, Value)>,
+    experiments: Vec<String>,
```

Comment:
This is such an esoteric field, I'd rather skip adding extra logic for it. We do not want this helper to grow infinitely. You can always call `manifest_extra` manually in tests needing it.

Reply:
hmm right. Perhaps it'd be better to have `manifest_package_extra` method?

Reply:
done, added that